gigascience/gigadb-website
# Remove old layouts

# Pull request for issue: #1892

This is a pull request for the following functionalities:

* Delete old layouts
* Rename new layouts by removing the prefix 'new'
* Delete column1 and column2 layouts as they serve no purpose
* Delete lines that set layout on a page basis if that is equivalent to the controller default
* Refactor of faq layout to use the header and footer partials
* Removed `type=text/javascript` from some script tags as it's not necessary

NOTE: I decided to push the deliverable about using one single layout for all pages and either views or partials for different features to another PR to avoid bloating this one

## How to test?

* No additional tests should fail as a consequence of this PR (one unit_functional_runner test has been failing now for some time)
* Optionally check that a subset of pages render as expected, e.g.
  * http://gigadb.gigasciencejournal.com/
  * http://gigadb.gigasciencejournal.com/dataset/100006
  * http://gigadb.gigasciencejournal.com/site/admin
  * http://gigadb.gigasciencejournal.com/adminDataset/admin
  * http://gigadb.gigasciencejournal.com/curationLog/update/id/3
  * http://gigadb.gigasciencejournal.com/site/faq

## Any issues with implementation?

--

## Any changes to automated tests?

--

## Any changes to documentation?

--

## Any technical debt repayment?

- This refactor should significatively reduce technical debt regarding maintaining the layout files
